R/render.r

```diff
@@ -162,6 +170,21 @@ check_open_graph <- function(og) {
   og[intersect(supported_fields, names(og))]
 }
 
+get_bs_version <- function(pkg = ".") {
+  pkg <- as_pkgdown(pkg)
+
+  template <- pkg$meta[["template"]]
+
+  if (!is.null(template$bootstrap)) {
+    if (!template$bootstrap %in% c(3, 4)) {
+      stop("The Bootstrap version `bootstrap` has to be 3 or 4.", call. = FALSE)
```

Comment:
Can you please follow https://style.tidyverse.org/error-messages.html ?

I think pkgdown uses rlang already, so please use `abort()` and then the error message should be a "must" statement, like "Boostrap version must be 3 or 4".